bom/deployment/pom.xml

```diff
@@ -748,6 +748,11 @@
                 <artifactId>quarkus-spring-cache-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-spring-scheduling-deployment</artifactId>
```

Comment:
This should probably be `scheduled` instead of `scheduling` because we use `quarkus-scheduled` in the existing extension.